content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "Installing Kubeflow"
+description = "Overview of installation choices for Kubeflow"
+weight = 20
+
++++
+
+Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
+Operators can choose what is best for their users, there is no requirement to deploy every component.
+To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/kubeflow-overview/">Kubeflow Overview</a> page.
+
+There are two pathways to get up and running with Kubeflow, you may either:
+1. Use a [packaged distribution](#packaged-distributions)
+1. Use the [manifests](#manifests) (advanced)
+
+<a id="packaged-distributions"></a>
+## Packaged distributions
+
+{{% alert color="warning" %}}
+Packaged distributions are developed and supported by their respective maintainers, the Kubeflow community does not currently endorse or certify any distribution.
+{{% /alert %}}
+
+<b>See the table below for a list of options and links to documentation:</b>
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Name</th>
+        <th>Maintainer</th>
+        <th>Platform</th>
+        <th>Link - Docs</th>
+        <th>Link - External Website</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Kubeflow on AWS</td>
+        <td>AWS</td>
+        <td>Amazon Elastic Kubernetes Service (EKS)</td>
+        <td><a href="/docs/distributions/aws/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on Azure</td>
+        <td>Azure</td>
+        <td>Azure Kubernetes Service (AKS)</td>
+        <td><a href="/docs/distributions/azure/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on GCP</td>
+        <td>Google Cloud</td>
```

Comment:
Google Cloud Platform (GCP)

Reply:
The google guys asked to make it Google Cloud (as that is the "entity" which is maintaining the distribution), @Bobgy thoughts?

Reply:
Yes, that's right. Now we stick to calling it Google Cloud.